great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -141,9 +137,9 @@ def __init__(
 
     def _init_rules(
```

Comment:
I think this could be a static method, the methods it depends (currently called through self) are also static.

This would make it easier to have init_rules live with rules rather than profilers

Reply:
@donaldheppner Good catch -- I made it static.